docs/3.api/3.utils/prerender-routes.md

```diff
@@ -28,3 +28,19 @@ prerenderRoutes(['/', '/about'])
 ::note
 In the browser, or if called outside prerendering, `prerenderRoutes` will have no effect.
 ::
+
+You can even prerender API routes which is particularly useful for full Statically Generated Sites (SSG) because you can then `$fetch` data as if you have an available server!
```

Comment:
```suggestion
You can even prerender API routes which is particularly useful for full statically generated sites (SSG) because you can then `$fetch` data as if you have an available server!
```